delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/model/ResourceRef.scala

```diff
@@ -13,7 +16,12 @@ sealed trait ResourceRef extends Product with Serializable {
     */
   def iri: Iri
```

Comment:
It is not clear when reading the comments the difference between an the iri and the original